ethcore/src/json_tests/test_common.rs

```diff
@@ -51,6 +52,8 @@ pub fn run_test_path<H: FnMut(&str, HookType)>(
 			}}) {
 			run_test_path(&p, skip, runner, start_stop_hook)
 		}
+	} else if extension == Some("swp") || extension == None {
+		// Ignore junk
```

Comment:
:(

Reply:
Yeah, there are some `swp` files or `.stub` files committed in the repo :(